lib/supply/options.rb

```diff
@@ -33,19 +33,18 @@ def self.available_options
                                      optional: true,
                                      description: "Path to the directory containing the metadata files",
                                      default_value: (Dir["./fastlane/metadata/android"] + Dir["./metadata"]).first),
-        FastlaneCore::ConfigItem.new(key: :key,
-                                     env_name: "SUPPLY_KEY",
+        # REVIEW: better name than :json ? :jsonsecret ?
+        # REVIEW: what about migration from a previous config ? is there a way to auto-deprecate options ?
```

Comment:
At some point it might be nice to have a `deprecated: true` argument to fastlane_core ConfigItem.